### ISSUE TITLE
Feature request: restore without init

### ISSUE THREAD

**Issue by nikjft** [gilbertchen/duplicacy#118]
It would be very useful to have a command-line option to restore a file without having to first init a repository. This would avoid all kinds of headaches.

**Comment by TheBestPessimist** [comment]
Not only a file, but to restore anything.

**Comment by gilbertchen** [comment]
I agree this can be convenient, but the implementation may be tricky, as the .duplicacy directory is used for several other things, like saving passwords or chunk caching.  So without a .duplicacy directory the code has to handle a few corner cases carefully.

**Comment by markfeit** [comment]
For restores, you can get away with creating a `.duplicacy` file that points at the existing `preferences` directory and Duplicacy will be none the wiser:

```
#!/bin/sh -e

PREFS_DIR=/path/to/duplicacy/preferences/directory
RESTORE_DIR=/path/where/restore/will/happen

# Prepare the restore directory
mkdir -p "{RESTORE_DIR}"
echo "${PREFS_DIR}" > "${RESTORE_DIR}/.duplicacy"

# Do the restore
cd "${RESTORE_DIR}"
duplicacy restore ...

# Clean up leftovers
rm -f "${RESTORE_DIR}/.duplicacy"
```

I do a superset of this in [duplicacy-scripts](https://github.com/markfeit/duplicacy-scripts) and it works just fine.
